rails/benchmarks/driver.rb

```diff
@@ -7,26 +7,19 @@
 require 'pathname'
 require 'optparse'
 require 'rails'
+require 'digest'
 
 RAW_URL = 'https://raw.githubusercontent.com/ruby-bench/ruby-bench-suite/master/rails/benchmarks/'
 
-sqlite3_url =
-  if Rails.version < '4.1.0'
-    'sqlite3:/:memory:'
-  else
-    'sqlite3::memory:'
-  end
-
 postgres_tcp_addr = ENV['POSTGRES_PORT_5432_TCP_ADDR'] || 'localhost'
 postgres_port = ENV['POSTGRES_PORT_5432_TCP_PORT'] || 5432
 mysql_tcp_addr = ENV['MYSQL_PORT_3306_TCP_ADDR'] || 'localhost'
 mysql_port = ENV['MYSQL_PORT_3306_TCP_PORT'] || 3306
 
-DATABASE_URLS = [
-  sqlite3_url,
```

Comment:
@kirs @SamSaffron I dropped support for sqlite3 here. I figured that the results are not really useful for majority of the Rails apps out there. Less benchmarks to run == backfill faster :stuck_out_tongue: